app.js

```diff
@@ -1,83 +1,83 @@
-let collection = [];
+// ----- LIBRARY CLASS -----
 
-// -----CREATE A FUNCTION WHICH ADDS A NEW OBJECT TO THE ARRAY-----
-// ARRAY SHOULD GET THE INFORMATION FROM THE FORM INPUT VALUE
-const bookData = document.querySelector('form');
-const addBtn = document.querySelector('.add__add-btn');
+class Library {
+  constructor(collection) {
+    this.collection = collection;
+  }
 
-const bookTemplate = document.querySelector('.books__template');
-const bookContainer = document.querySelector('.container');
+  addObjectToArray() {
+    const bookData = document.querySelector('form');
+    const title1 = bookData.book_title.value;
+    const author1 = bookData.book_author.value;
+
+    const book = {
+      title: title1,
+      author: author1,
+    };
+    this.collection.push(book);
```

Comment:
[Optional]
You should use array methods that doesn't mutate the array, e.g. [`concat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [`filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [`slice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [`push`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [`pop`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [`splice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice).

Mutation hides change, which creates unexpected side effects, which can cause nasty bugs. When you enforce immutability you can keep your application architecture and mental model simple, which makes it easier to reason about your application